packages/thirdweb/src/wallets/manager/index.ts

```diff
@@ -121,9 +114,6 @@ export function createConnectionManager(storage: AsyncStorage) {
 
     handleSetActiveWallet(activeWallet);
 
-    // add personal wallet to connected wallets list
-    addConnectedWallet(personalWallet);
```

Comment:
I think we should still add the EOA as a connected wallet. What's the reasoning for removing this?

Reply:
reverted this change